scripts/create-hugo-site

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+#
+# Creates pre-requiste hugo site
+set -eu
+
+readonly PROGNAME=$(basename "$0")
+readonly ARGS="$@"
+APP_NAME=
```

Comment:
I'm forcing the user to provide that via the command line option. Thinking more about it, I'll make it `APP_NAME=app` and relax the app name option.